com.unity.shadergraph/Editor/Data/Nodes/Procedural/CheckerboardNode.cs

```diff
@@ -27,12 +27,12 @@ protected override MethodInfo GetFunctionToConvert()
             return
 @"
 {
-    UV = (UV.xy + 0.5) * Frequency;
-    $precision2 distance3 = 4.0 * abs(frac(UV + 0.25) - 0.5) - 1.0;
 #if defined(SHADER_STAGE_RAY_TRACING)
-    int2 alpha = saturate(distance3 * FLT_MAX);
```

Comment:
this must be throwing warnings on console

Reply:
Well, it's gone now.

Reply:
Ah, no it doesn't.